modules/products/bitbucket/nfs/variables.tf

```diff
@@ -0,0 +1,22 @@
+variable "product" {
+  description = "The product that NFS will be created for."
+  type        = string
+}
+
+variable "namespace" {
+  description = "Kubernetes namespace to install Bitbucket."
+  type        = string
+}
+
+variable "chart_name" {
+  description = "The chart name to use."
```

Comment:
It is not name of the chart, is it? It is used in the `values` so I assume it changes the helm chart behaviour. Does it change the `helm release name`?

Reply:
So we pass this to have control over the name of the release and resultant pod names. By default this would be `nfs-server-example`. Through this variable we override the name so that it is now `bitbucket-nfs-server`

Reply:
It might not be necessary to do this though - maybe we can live with it being called `nfs-server-example` and hence remove the variable?

Reply:
No, I think it is a good idea to override it.